README.md

```diff
@@ -49,7 +49,14 @@ To whitelist your pages:
   tabs opened will be synchronized.  This means that state changes made in one
   open window will be communicated to all open windows.
 
-### Downloading Streams
+### Downloading Streams with npm
+`npm install amazon-connect-streams`
+
+#Importing Streams with npm and ES6
```

Comment:
Did you mean for this to be a header? if so it should be # Importing

Reply:
Good point! Fixed